tdameros/42-transcendence
# Made a server that redirects user to another server

To test:
1. Create a venv in pong-server using pong-server/docker/requirements.txt
2. run `cd pong-server && rerun.sh`

4. go to front/src/threejs
5. run `npm i`
6. run `npx vite`
7. open the link shown by `npx vite` on a browser

Open the browser console to see the logs
